### ISSUE TITLE
chado 1.3 install issues

### ISSUE THREAD

**Issue by bradfordcondon** [statonlab/hardwoods_site#268]
Noticed this because my organism linker field is broke for featuremap on dev because theres no table!

### 1.31 tables present
* contactprop
* featuremap_dbxref
* featuremapprop
*  featureposprop 
* pubauthor_contact

### 1.31 table absent
* analysis_cvterm
* analysis_dbxref
* analysis_pub
* dbprop
* feature_contact
* featuremap_contact
* featuremap_organism
* library_contact
* library_expression
* library_expressionprop
* library_featureprop
* library_relationship
* library_relationship_pub
* nd_experiment_analysis
* organism_cvterm
* organism_cvtermprop
* organism_pub
* organism_relationship
* organismprop_pub
* phenotypeprop
* phylotreeprop
* project_analysis
* project_dbxref
* project_feature
* Project_stock
* stock_feature
* stock_featuremap
* stock_library
* stockcollection_db

The create tables from 1.2-1.31 sql is below.

```
CREATE TABLE analysis_cvterm (
CREATE TABLE analysis_dbxref (
CREATE TABLE analysis_pub (
CREATE TABLE analysis_relationship (
CREATE TABLE contactprop (
CREATE TABLE dbprop (
CREATE TABLE feature_contact (
CREATE TABLE featuremap_contact (
CREATE TABLE featuremap_dbxref (
CREATE TABLE featuremap_organism (
CREATE TABLE featuremapprop (
CREATE TABLE featureposprop (
CREATE TABLE library_contact (
CREATE TABLE library_expression (
CREATE TABLE library_expressionprop (
CREATE TABLE library_featureprop (
CREATE TABLE library_relationship (
CREATE TABLE library_relationship_pub (
CREATE TABLE nd_experiment_analysis (
CREATE TABLE organism_cvterm (
CREATE TABLE organism_cvtermprop (
CREATE TABLE organism_pub (
CREATE TABLE organism_relationship (
CREATE TABLE organismprop_pub (
CREATE TABLE phenotypeprop (
CREATE TABLE phylotreeprop (
CREATE TABLE project_analysis (
CREATE TABLE project_dbxref (
CREATE TABLE project_feature (
CREATE TABLE project_stock (
CREATE TABLE pubauthor_contact (
CREATE TABLE stock_feature (
CREATE TABLE stock_featuremap (
CREATE TABLE stock_library (
CREATE TABLE stockcollection_db (
````

**Comment by bradfordcondon** [comment]
note: this is from the 1.23-1.31 diff.

**Comment by bradfordcondon** [comment]
my dev:
![screen shot 2018-05-08 at 10 45 54 am](https://user-images.githubusercontent.com/7063154/39764199-1764d16e-52ad-11e8-88ff-a757656f0b76.png)

server dev:
![screen shot 2018-05-08 at 10 44 43 am](https://user-images.githubusercontent.com/7063154/39764203-18d9509c-52ad-11e8-9eb6-d0c7b9b5a32a.png)

Its a mystery why we have a few 1.31 tables.  but what is clear is that we dont use chado 1.3 installed at least not complete.

**Comment by bradfordcondon** [comment]
- [ ] PostgreSQL version 9.1 is required to perform this upgrade. If your Tripal site uses an older version please upgrade before proceeding.

- [ ] A major change between Chado v1.2 and v1.3 is that primary and foreign keys were upgraded from integers to big integers. If your site has custom materialized views that will hold data derived from fields changed to big integers then you may need to alter the views to change the fields from integers to big integers and repopulate those views. If you have not added any materialized views you can ignore this issue.

- [ ] Custom PL/pgSQL functions that expect primary and foreign key fields to be integers will not work after the upgrade. Those functions will need to be altered to accept big integers. If you do not have any custom PL/pgSQL functions you can ignore this issue.

- [ ] PostgreSQL Views that use fields that are converted to big integers will cause this upgrade to fail. You must first remove those views, perform the upgrade and then recreate them with the appropriate fields change to big integers. If you do not have custom PostgreSQL Views you can ignore this issue.

- [ ] Several new tables were added to Chado v1.3. However, some groups have added these tables to their Chado v1.2 installation. The Tripal upgrader will alter the primary and foreign keys of those tables to be "bigints" if they already exist but will otherwise leave them the same. You should verify that any tables with Chado v1.3 names correctly match the v1.3 schema. Otherwise you may have problems using Tripal. If you have not added any Chado v1.3 tables to your Chado v1.2 database you can ignore this issue. These are the newly added tables: analysis_cvterm, analysis_dbxref, analysis_pub, analysis_relationship, contactprop, dbprop, feature_contact, featuremap_contact, featuremap_dbxref, featuremap_organism, featuremapprop, featureposprop, library_contact, library_expression, library_expressionprop, library_featureprop, library_relationship, library_relationship_pub, nd_experiment_analysis, organism_cvterm, organism_cvtermprop, organism_pub, organism_relationship, organismprop_pub, phenotypeprop, phylotreeprop, project_analysis, project_dbxref, project_feature, project_stock, pubauthor_contact, stock_feature, stock_featuremap, stock_library, stockcollection_db.

**Comment by bradfordcondon** [comment]
upgrade fails with 
```
                                                                                   [error]
SQLSTATE[0A000]: Feature not supported: 7 ERROR:  cannot alter type of a column used by a view or rule
DETAIL:  rule _RETURN on view so.unorient_inter_transposition depends on column "feature_id":
ALTER TABLE feature
    ALTER feature_id TYPE  bigint,
    ALTER dbxref_id TYPE bigint,
    ALTER organism_id TYPE bigint,
    ALTER seqlen TYPE bigint,
    ALTER type_id TYPE bigint;



[site http://160.36.205.61:8095] [TRIPAL ERROR] [TRIPAL_CHADO] FAILED. Line  2374, 0SQLSTATE[0A000]: Feature not supported: 7 ERROR:  cannot alter type of a column used by a view or ruleDETAIL:  rule _RETURN on view so.unorient_inter_transposition depends on column "feature_id":ALTER TABLE feature     ALTER feature_id TYPE  bigint,    ALTER dbxref_id TYPE bigint,    ALTER organism_id TYPE bigint,    ALTER seqlen TYPE bigint,    ALTER type_id TYPE bigint;
```

we need to find our views and delete or upgrade.

**Comment by bradfordcondon** [comment]
The SO views are problematic.  They also are obsolete, orat the least, tripal doesnt use them at all, so we are goign to drop them.  some get re-added in the 1.2-1.3 diff.  The problematic ones are defined in 1.1 and not dropped in the 1.2 diff.

http://gmod.org/wiki/Chado_SO_Views

**Comment by bradfordcondon** [comment]
```select 'drop view "' || viewname || '" cascade;' 
from pg_views where schemaname = 'so';```

to generate drop script

**Comment by bradfordcondon** [comment]
we also had to drop all public views.

```
: Feature not supported: 7 ERROR:  cannot alter type of a column used by a view or rule
DETAIL:  rule _RETURN on view public.cvterm depends on column "cvterm_id":
ALTER TABLE cvterm
    ALTER cvterm_id TYPE bigint,
    ALTER cv_id TYPE bigint,
    ALTER dbxref_id TYPE bigint;



[site http://160.36.205.61:8095] [TRIPAL ERROR] [TRIPAL_CHADO] FAILED. Line  2469, 0SQLSTATE[0A000]: Feature not supported: 7 ERROR:  cannot alter type of a column used by a view or ruleDETAIL:  rule _RETURN on view public.cvterm depends on column "cvterm_id":ALTER TABLE cvterm    ALTER cvterm_id TYPE bigint,    ALTER cv_id TYPE bigint,    ALTER dbxref_id TYPE bigint;
```

**Comment by almasaeed2010** [comment]
To delete in public has to be done in psql

```sql
drop view public.orgnasim;
drop view public.organism;
drop view public.cvterm;
drop view public.feature_ssr_mview;
```

**Comment by bradfordcondon** [comment]
https://github.com/tripal/tripal/issues/372

**Comment by bradfordcondon** [comment]
- [ ] upgrade dev
``Upgrade from v1.2 to v1.3 Successful!``

- [ ] upgrade live

**Comment by almasaeed2010** [comment]
The deadlock issue can be solved by doing the following
- open `sites/all/modules/tripal/tripal_chado/api/tripal_chado.schema.api.in`
- comment our like 297 and 301 (the lines the have `variable_set('chado_schema_exists', FALSE);`
- save

**Comment by bradfordcondon** [comment]
ran out of disk space live?

```
Calling: tripal_chado_install_chado(Upgrade Chado v1.2 to v1.3)
Checking for existing v1.3 tables in v1.2 and fixing bigints...
Incorporating additional changes...
Loading sites/all/modules/tripal/tripal_chado/chado_schema/default_schema-1.2-1.3-diff.sql...
WD tripal_chado: FAILED. Line  2374, 0                               [error]
SQLSTATE[53100]: Disk full: 7 ERROR:  could not extend file
"base/10057741/13327847.43": No space left on device
HINT:  Check free disk space.:
ALTER TABLE feature
    ALTER feature_id TYPE  bigint,
    ALTER dbxref_id TYPE bigint,
    ALTER organism_id TYPE bigint,
    ALTER seqlen TYPE bigint,
    ALTER type_id TYPE bigint;



[site http://default] [TRIPAL ERROR] [TRIPAL_CHADO] FAILED. Line  2374, 0SQLSTATE[53100]: Disk full: 7 ERROR:  could not extend file "base/10057741/13327847.43": No space left on deviceHINT:  Check free disk space.:ALTER TABLE feature     ALTER feature_id TYPE  bigint,    ALTER dbxref_id TYPE bigint,    ALTER organism_id TYPE bigint,    ALTER seqlen TYPE bigint,    ALTER type_id TYPE bigint;
[bcondon@hardwoodgenomics ~]$
```

**Comment by bradfordcondon** [comment]
featuremap_dbxref is missing the is_current column.

I added it manually on dev.  Need to do so on live too.


However this raises the possibility that the following tables are also bugged in some way:

* contactprop
* featuremap_dbxref
* featuremapprop
* featureposprop
* pubauthor_contact

**Comment by bradfordcondon** [comment]
- [ ] Add `is_current` column to freaturemap_dbxref on live

**Comment by bradfordcondon** [comment]
`ALTER TABLE featuremap_dbxref ADD COLUMN is_current bool;`

**Comment by bradfordcondon** [comment]
closing as resolved for now.  as mentioned above, some of the other pre-existing tables might have problems down the road...

**Comment by bradfordcondon** [comment]
missing `featuremap_organism` on live, and the version says 1.2.  we didnt finish migrating on live!

**Comment by bradfordcondon** [comment]
we need to disable transaction (see below code) for the chado migrate to not run otu of space.  thats risky, so we're backing up first.  

```
$transaction = db_transaction();
  try {
    if ($action == 'Install Chado v1.3') {
      tripal_chado_install_chado_1_3();
      chado_query($vsql, array(':version' => '1.3'));
    }
    elseif ($action == 'Upgrade Chado v1.2 to v1.3') {
      tripal_chado_upgrade_chado_1_2_to_1_3();
      chado_query($vusql, array(':version' => '1.3'));
    }
    elseif ($action == 'Install Chado v1.2') {
      tripal_chado_install_chado_1_2();
      chado_query($vsql, array(':version' => '1.2'));
    }
    elseif ($action == 'Upgrade Chado v1.11 to v1.2') {
      tripal_chado_upgrade_chado_1_11_to_1_2();
      chado_query($vsql, array(':version' => '1.2'));
    }
    elseif ($action == 'Install Chado v1.11') {
      tripal_chado_install_chado_1_11();
    }
  }
  catch (Exception $e) {
    $transaction->rollback();
    tripal_chado_install_done();
    tripal_report_error('tripal_chado', TRIPAL_ERROR, $e->getMessage(), array('print' => TRUE));
    return FALSE;
  }
```

**Comment by bradfordcondon** [comment]
running again after disabling transactions and backing up:

```

2018-05-31 16:03:44: There are 1 jobs queued.
2018-05-31 16:03:44: Calling: tripal_chado_install_chado(Upgrade Chado v1.2 to v1.3)
Checking for existing v1.3 tables in v1.2 and fixing bigints...
Incorporating additional changes...
Loading sites/all/modules/tripal/tripal_chado/chado_schema/default_schema-1.2-1.3-diff.sql...
WD tripal_chado: FAILED. Line  2374, 0                                                                                          [error]
SQLSTATE[53100]: Disk full: 7 ERROR:  could not extend file "base/10057741/13914652.42": wrote only 4096 of 8192 bytes at block
5581190
HINT:  Check free disk space.:
ALTER TABLE feature
    ALTER feature_id TYPE  bigint,
    ALTER dbxref_id TYPE bigint,
    ALTER organism_id TYPE bigint,
    ALTER seqlen TYPE bigint,
    ALTER type_id TYPE bigint;



[site http://default] [TRIPAL ERROR] [TRIPAL_CHADO] FAILED. Line  2374, 0SQLSTATE[53100]: Disk full: 7 ERROR:  could not extend file "base/10057741/13914652.42": wrote only 4096 of 8192 bytes at block 5581190HINT:  Check free disk space.:ALTER TABLE feature     ALTER feature_id TYPE  bigint,    ALTER dbxref_id TYPE bigint,    ALTER organism_id TYPE bigint,    ALTER seqlen TYPE bigint,    ALTER type_id TYPE bigint
```

**Comment by bradfordcondon** [comment]
after freeing up lots of disk space, and applying the deadlock fix:

```
[bcondon@hardwoodgenomics ~]$ drush trp-run-jobs --username=hwadmin --root=/var/www/html

2018-06-04 10:09:40
Tripal Job Launcher
Running as user 'hwadmin'
-------------------
2018-06-04 10:09:40: There are 1 jobs queued.
2018-06-04 10:09:40: Calling: tripal_chado_install_chado(Upgrade Chado v1.2 to v1.3)
Checking for existing v1.3 tables in v1.2 and fixing bigints...
Incorporating additional changes...
Loading sites/all/modules/tripal/tripal_chado/chado_schema/default_schema-1.2-1.3-diff.sql...
WD tripal_chado: FAILED. Line  3443, 0                                                                                           [error]
SQLSTATE[40P01]: Deadlock detected: 7 ERROR:  deadlock detected
DETAIL:  Process 20554 waits for AccessExclusiveLock on relation 14142549 of database 14142031; blocked by process 21062.
Process 21062 waits for AccessShareLock on relation 14142179 of database 14142031; blocked by process 20554.
HINT:  See server log for query details.:
ALTER TABLE chadoprop
    ALTER chadoprop_id TYPE bigint,
    ALTER type_id TYPE bigint;



[site http://default] [TRIPAL ERROR] [TRIPAL_CHADO] FAILED. Line  3443, 0SQLSTATE[40P01]: Deadlock detected: 7 ERROR:  deadlock detectedDETAIL:  Process 20554 waits for AccessExclusiveLock on relation 14142549 of database 14142031; blocked by process 21062.Process 21062 waits for AccessShareLock on relation 14142179 of database 14142031; blocked by process 20554.HINT:  See server log for query details.:ALTER TABLE chadoprop     ALTER chadoprop_id TYPE bigint,    ALTER type_id TYPE bigint;
[bcondon@hardwoodgenomics ~]$
```

we're not sure how to proceed here...

**Comment by bradfordcondon** [comment]
we transferred the db to dev to rule out hardware issues.

```
Calling: tripal_chado_install_chado(Upgrade Chado v1.2 to v1.3)
Checking for existing v1.3 tables in v1.2 and fixing bigints...
Incorporating additional changes...
Loading sites/all/modules/tripal/tripal_chado/chado_schema/default_schema-1.2-1.3-diff.sql...
WD tripal_chado: FAILED. Line  3443, 0                                                                                          [error]
SQLSTATE[40P01]: Deadlock detected: 7 ERROR:  deadlock detected
DETAIL:  Process 24729 waits for AccessExclusiveLock on relation 47987596 of database 47986142; blocked by process 24983.
Process 24983 waits for AccessShareLock on relation 47987225 of database 47986142; blocked by process 24729.
HINT:  See server log for query details.:
ALTER TABLE chadoprop
    ALTER chadoprop_id TYPE bigint,
    ALTER type_id TYPE bigint;



[site http://160.36.205.61:8095] [TRIPAL ERROR] [TRIPAL_CHADO] FAILED. Line  3443, 0SQLSTATE[40P01]: Deadlock detected: 7 ERROR:  deadlock detectedDETAIL:  Process 24729 waits for AccessExclusiveLock on relation 47987596 of database 47986142; blocked by process 24983.Process 24983 waits for AccessShareLock on relation 47987225 of database 47986142; blocked by process 24729.HINT:  See server log for query details.:ALTER TABLE chadoprop     ALTER chadoprop_id TYPE bigint,    ALTER type_id TYPE bigint;
```

again with the chadoprop alter statement and deadlocks.  So at this point seems very likely its something specific to the live db related to the prop tables.

**Comment by bradfordcondon** [comment]
I manually re-ran the failing line, deleted from the diff, and re-ran the migration job. 

Failing line:

```
ALTER TABLE chadoprop 
    ALTER chadoprop_id TYPE bigint,
    ALTER type_id TYPE bigint;
    
```

It ran with no errors.

**Comment by bradfordcondon** [comment]
now we just fail here:
```
SQLSTATE[0A000]: Feature not supported: 7 ERROR:  cannot alter type of a column used by a view or rule
DETAIL:  rule _RETURN on view public.library depends on column "library_id":
ALTER TABLE library
    ALTER library_id TYPE bigint,
    ALTER organism_id TYPE bigint,
    ALTER type_id TYPE bigint;



[site http://160.36.205.61:8095] [TRIPAL ERROR] [TRIPAL_CHADO] FAILED. Line  4137, 0SQLSTATE[0A000]: Feature not supported: 7 ERROR:  cannot alter type of a column used by a view or ruleDETAIL:  rule _RETURN on view public.library depends on column "library_id":ALTER TABLE library     ALTER library_id TYPE bigint,    ALTER organism_id TYPE bigint,    ALTER type_id TYPE bigint;
```

OK... lets try splitting up the sql file and running it up until that line.

**Comment by bradfordcondon** [comment]
i cant drop the view?  via drush sql-cli.  i bet the databases have different owners.
```
drop view public.library;
ERROR:  must be owner of relation library
```

**Comment by bradfordcondon** [comment]
```
                     [error]
SQLSTATE[42704]: Undefined object: 7 ERROR:  constraint "organism_c1" of relation "organism" does not exist:
ALTER TABLE ONLY organism DROP CONSTRAINT organism_c1;



[site http://160.36.205.61:8095] [TRIPAL ERROR] [TRIPAL_CHADO] FAILED. Line  6464, 0SQLSTATE[42704]: Undefined object: 7 ERROR:  constraint "organism_c1" of relation "organism" does not exist:ALTER TABLE ONLY organism DROP CONSTRAINT organism_c1;
```

all the other constrain drops have an IF EXISTS.  adding one to this statement and rerunning.

**Comment by bradfordcondon** [comment]
next error:

```
FAILED. Line  7614, 0                                                                                                             [error]
SQLSTATE[42830]: Invalid foreign key: 7 ERROR:  there is no unique constraint matching given keys for referenced table "pub":
ALTER TABLE ONLY analysis_pub
    ADD CONSTRAINT analysis_pub_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;



[site http://160.36.205.61:8095] [TRIPAL ERROR] [TRIPAL_CHADO] FAILED. Line  7614, 0SQLSTATE[42830]: Invalid foreign key: 7 ERROR:  there is no unique constraint matching given keys for referenced table "pub":ALTER TABLE ONLY analysis_pub    ADD CONSTRAINT analysis_pub_pub_id_fkey FOREIGN KEY (pub_id) REFERENCES pub(pub_id) ON DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;
```

at this point it's clear we are getting an error once every 1k-2k lines of the diff.  its 50k lines long.

**Comment by bradfordcondon** [comment]
theres no explanation for why our chado.pub table does not have constrains/primary keys defined.

We recognize that fixing each problem manually will take days (weeks?).  And that even getting the migration to work wont necessarily fix all hte problems, just those that arise when upgrading from 1.2 to 1.3
* fix chado on per-error basis trying ot migrate
* Migrate dev to live.  This will break lots of things.  mail module, blast locations, all sorts of files that will have different absolute paths...
* dump data from live (DATA only, not STRUCTURE).  make a fresh site, from chado 1.3  Then import the data.

**Comment by bradfordcondon** [comment]
Plan to do the "migrate, fix migrate"
start of each file: `set search_path=public,so,frange,genetic_code;`

**Comment by bradfordcondon** [comment]
```WD tripal_chado: FAILED. Line  7685, 0                                                                                                             [error]
SQLSTATE[42830]: Invalid foreign key: 7 ERROR:  there is no unique constraint matching given keys for referenced table "feature":
ALTER TABLE ONLY feature_contact
    ADD CONSTRAINT feature_contact_feature_id_fkey FOREIGN KEY (feature_id) REFERENCES feature(feature_id) ON DELETE CASCADE;



[site http://160.36.205.61:8095] [TRIPAL ERROR] [TRIPAL_CHADO] FAILED. Line  7685, 0SQLSTATE[42830]: Invalid foreign key: 7 ERROR:  there is no unique constraint matching given keys for referenced table "feature":ALTER TABLE ONLY feature_contact    ADD CONSTRAINT feature_contact_feature_id_fkey FOREIGN KEY (feature_id) REFERENCES feature(feature_id) ON DELETE CASCADE;
[bcondon@localhost chado_schema]$
```

**Comment by bradfordcondon** [comment]
so clearly i made my truncation a few lines too early and im paying for it.


```
WD tripal_chado: FAILED. Line  7825, 0                                                                                                             [error]
SQLSTATE[42830]: Invalid foreign key: 7 ERROR:  there is no unique constraint matching given keys for referenced table "library_feature":
ALTER TABLE ONLY library_featureprop
    ADD CONSTRAINT library_featureprop_library_feature_id_fkey FOREIGN KEY (library_feature_id) REFERENCES library_feature(library_feature_id) ON
DELETE CASCADE DEFERRABLE INITIALLY DEFERRED;

```

**Comment by bradfordcondon** [comment]
```WD tripal_chado: SQLSTATE[42P07]: Duplicate table: 7 ERROR:  relation "analysis_cvterm_analysis_cvterm_id_seq" already exists                      [error]
[site http://160.36.205.61:8095] [TRIPAL ERROR] [TRIPAL_CHADO] SQLSTATE[42P07]: Duplicate table: 7 ERROR:  relation "analysis_cvterm_analysis_cvterm_id_seq" already exists
```
next part... but this table isnt in the sql file...

**Comment by bradfordcondon** [comment]
add p-keys to all tables.  
```
`SQLSTATE[42P01]: Undefined table: 7 ERROR:  relation "featureprop_c1" does not exist:
COMMENT ON INDEX featureprop_c1 IS 'For any one feature, multivalued
property-value pairs must be differentiated by rank.';
```
made it to `44390` of the second file.

**Comment by bradfordcondon** [comment]
```
 Calling: tripal_chado_install_chado(Upgrade Chado v1.2 to v1.3)
Checking for existing v1.3 tables in v1.2 and fixing bigints...
Incorporating additional changes...
Loading sites/all/modules/tripal/tripal_chado/chado_schema/default_schema-1.2-1.3-diff.sql...
Upgrade from v1.2 to v1.3 Successful!
```

:O  I think that does it!  

Unfortunately abdullah is out today so i dont know that we'll migrate the dev site back to live today...

**Comment by bradfordcondon** [comment]
creating a dump of the hardwoods_live_replica db, and will transfer it to live.  Once it is finished ill transfer dev back to the original db so people can continue working on dev. I'm thinking this is just done in settings.php?

creating the dump:
```
pg_dump hardwoods_live_replica > hwg_live_06_08_18.dump
```

**Comment by bradfordcondon** [comment]
```
pg_restore -U hardwoods -d hardwoods_06_08_2018 -1 hwg_live_06_08_18.dump
pg_restore: [archiver] input file appears to be a text format dump. Please use psql.
```

oops i guess i needed to use some different flags creating the dump to use pg_restore, i need to use psql instead.

but this command just vies me a string of warnings?
`sudo psql -U hardwoods -d hardwoods_06_08_2018 -1 -f hwg_live_06_08_18.dump`

**Comment by almasaeed2010** [comment]
Did this pass? Should we create a compressed dump for the DB since we have limited space on live?

**Comment by bradfordcondon** [comment]
hey.  no it didnt, so i deleted that dump and  reran as `sudo -u postgres pg_dump -Fc hardwoods_live_replica > Jun082018live.dump`

**Comment by bradfordcondon** [comment]
```
pg_restore -U hardwoods -d hardwoods_06_08_2018 -1 Jun082018live.dump
pg_restore: [archiver] unsupported version (1.13) in file header
```

**Comment by almasaeed2010** [comment]
I don't think you need `-1` there. Let me try restore again on live

**Comment by almasaeed2010** [comment]
Same result with or without `-1` :(

**Comment by almasaeed2010** [comment]
I don't know why this got closed lol..


I am currently redoing the dump (just in case) and gonna try to restore again.

**Comment by bradfordcondon** [comment]
1.3 installed live!

![screen shot 2018-06-11 at 12 42 17 pm](https://user-images.githubusercontent.com/7063154/41245056-f4042ade-6d74-11e8-9509-14b167a852c0.png)
